sympy/printing/tests/test_str.py

```diff
@@ -660,14 +660,14 @@ def test_settings():
 def test_RandomDomain():
     from sympy.stats import Normal, Die, Exponential, pspace, where
     X = Normal('x1', 0, 1)
-    assert str(where(X > 0)) == "Domain: And(0 < x1, x1 < oo)"
+    assert str(where(X > 0)) == "Domain: (0 < x1 & x1 < oo)"
 
     D = Die('d1', 6)
-    assert str(where(D > 4)) == "Domain: Or(Eq(d1, 5), Eq(d1, 6))"
+    assert str(where(D > 4)) == "Domain: (Eq(d1, 5) | Eq(d1, 6))"
 
     A = Exponential('a', 1)
     B = Exponential('b', 1)
-    assert str(pspace(Tuple(A, B)).domain) == "Domain: And(0 <= a, 0 <= b, a < oo, b < oo)"
+    assert str(pspace(Tuple(A, B)).domain) == "Domain: (0 <= a & 0 <= b & a < oo & b < oo)"
```

Comment:
I assume you used the ``precedence`` function, which follows this precedence table:
https://github.com/sympy/sympy/blob/master/sympy/printing/precedence.py#L8

Do we really need parentheses around the inequalities in that case? Don't relational operators have precedence over logical ``And``? (at least from a programming language perspective)

However, this does look more appealing:
```python
"Domain: (0 <= a) & (0 <= b) & (a < oo) & (b < oo)"
```